TIPS.md

```diff
@@ -6,26 +6,49 @@ Instructions for play
+- To discard cards, right click and select discard. Reclaim, by selecting the card(s) from the discard zone (below your cards in hand area) and drag them up.
+
 Intermediate Tips:
 - Press numpad keys (as shown in the bottom right) to spawn new pieces
   - Make sure numlock is enabled
   - Note that these scripting keys can be rebound in the TTS settings
-- Lock pieces (right-click and select lock)
+- Lock pieces (hover over the piece and right-click->Toggles->Lock or press "L")
 - Mark damage on pieces by changing the state
   - i.e. to mark one damage on a town, right click the town and change the state to 2
   - You can also press a number key to change the state (so press 2 to change the state to 2)
   - Note that invaders that have extra health or deal extra damage are rendered in a different color (i.e. red for england buildings, and blue for russian explorers)
 - Defend values can also be set as a state
 - To prevent a piece from going away during time passes you should lock it (right-click and select lock)
 - You can use energy tokens instead of the counter by deleting the energy counter (right-click on the energy counter and press delete)
+  - Note that deleting requires you to be the host or to be promoted
+- When removing blight from the blight card (usually for an event), move it to the "box" to the top-left of the play area (instead of deleting it)
+- When you have dealt enough damage to an invader to destroy it, drag it from the island board onto the "play mat" and add the corresponding amount of fear to the fear pool. 
+  - This has to be manual because some events change the amount of fear that you generate
   
 Advanced Tips:
 - UI elements can be enabled and disabled with the little triangle near the top-right of your screen
 - You can play double-handed (i.e. play 2 or more spirits) by changing your TTS player color in the top-right of the screen and choosing another spirit
   - Note that your UI elements are tracked per spirit
-- You can bind keys to many other actions with options->keys (TODO check name)
-- When playing as Shroud of Silent Mist it's important to not heal invaders and dahan in its land when time passes
-  - You can do this by locking the bottom invader in a stack or by stacking the damaged invader on top of a scenario token
+- You can bind keys to many other actions with Options (near the top-middle of the screen) -> Game Keys
 - If you play thresholded "Cast Down Into the Briny Deeps" use the gizmo tool (one of the tools on the left side of the screen) to move the island board
 - You can choose what island boards to play with at setup by placing the island boards before starting your game
 - To change what spirit you're playing after you've already started the game either reload the mod to restart everything, or unlock the spirit panel (right-click and press lock) and delete the spirit panel and any extra tokens/cards it has (right-click and select delete), then drag over a new spirit panel.
```

Comment:
Deleting a board and dragging over another one isn't really an intended feature, so maybe just mention that you should reload

Reply:
yea same i agree, should probably just mention reloading the mod